pkgs/applications/misc/streamdeck-ui/default.nix

```diff
@@ -74,33 +94,41 @@ python3Packages.buildPythonApplication rec {
     wrapGAppsHook3
   ];
 
-  propagatedBuildInputs = with python3Packages; [
-    setuptools
-    filetype
-    cairosvg
-    pillow
-    pynput
-    pyside6
-    streamdeck
-    xlib
-  ] ++ lib.optionals stdenv.isLinux [
-    qt6.qtwayland
-  ];
+  propagatedBuildInputs =
+    with python3Packages;
+    [
+      setuptools
+      filetype
+      cairosvg
+      pillow
+      pynput
+      pyside6
+      streamdeck
+      xlib
+      importlib-metadata_6
+    ]
```

Comment:
Would `evdev` dependency be missing?

https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/blob/main/pyproject.toml#L22C1-L22C6

https://pypi.org/project/evdev/

Reply:
Added, thanks